From 34bbdc1af3d76f556e26ac92fff35bf2ac853470 Mon Sep 17 00:00:00 2001
Message-Id: <34bbdc1af3d76f556e26ac92fff35bf2ac853470.1576692900.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Wed, 18 Dec 2019 10:02:59 -0800
Subject: [PATCH] notes.h: fix typos in comment
To: Git Mailing List <git@vger.kernel.org>
Cc: Eric Sunshine <sunshine@sunshineco.com>,
    Elijah Newren <newren@gmail.com>,
    Philip Oakley <philipoakley@iee.email>

In 1d7297513d (notes: break set_display_notes() into smaller functions,
2019-12-11), we introduced a comment which had a couple of typos. In the
first typo, we referenced 'enable_default_display_notes' instead of
'enable_ref_display_notes'. In the second typo, we wrote "is a points to"
instead of "is a pointer to". Correct both of these typos.

Reported-by: Eric Sunshine <sunshine@sunshineco.com>
Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
This patch is based on top of 'dl/format-patch-notes-config-fixup'.
Thanks for catching the typo, Eric!

 notes.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/notes.h b/notes.h
index 3e78448181..bbab1961ca 100644
--- a/notes.h
+++ b/notes.h
@@ -268,11 +268,11 @@ void init_display_notes(struct display_notes_opt *opt);
 /*
  * This family of functions enables or disables the display of notes. In
  * particular, 'enable_default_display_notes' will display the default notes,
- * 'enable_default_display_notes' will display the notes ref 'ref' and
+ * 'enable_ref_display_notes' will display the notes ref 'ref' and
  * 'disable_display_notes' will disable notes, including those added by previous
  * invocations of the 'enable_*_display_notes' functions.
  *
- * 'show_notes' is a points to a boolean which will be set to 1 if notes are
+ * 'show_notes' is a pointer to a boolean which will be set to 1 if notes are
  * displayed, else 0. It must not be NULL.
  */
 void enable_default_display_notes(struct display_notes_opt *opt, int *show_notes);
-- 
2.24.1.703.g2f499f1283

